src/rgw/rgw_file.h

```diff
@@ -1191,10 +1194,13 @@ namespace rgw {
 			    RGWFileHandle::FHCache::FLAG_LOCK);
       /* LATCHED */
       if (! fh) {
+	if (unlikely(fhk == root_fh.fh.fh_hk)) {
```

Comment:
what's different from the following code（line1127）

Reply:
@yangruifeng I don't entirely understand your question, but this is a backport so the similarity with the code in master is intentional.

Reply:
@smithfarm it's ok,  I just think it's a little redundant